### ISSUE TITLE
Refresh BALSE

### ISSUE THREAD

**Issue by jilljenn** [mangaki/zero#15]
- [ ] Compare various hyper-parameters of ElasticNet (notably on some test users to check convergence)
- [ ] Possibly reimplement with Keras
- [ ] Plot 3D curve with the loss (for β, γ)
- [ ] Put it in zero
